src/test/regress/sql/multi_extension.sql

```diff
@@ -220,6 +229,15 @@ SELECT * FROM multi_extension.print_extension_changes();
 ALTER EXTENSION citus UPDATE TO '9.5-1';
 SELECT * FROM multi_extension.print_extension_changes();
 
+-- Test upgrade paths for backported improvement of master_update_table_statistics function
+ALTER EXTENSION citus UPDATE TO '9.5-3';
+ALTER EXTENSION citus UPDATE TO '9.5-2';
+-- Should be empty result
+SELECT * FROM multi_extension.print_extension_changes();
+
+ALTER EXTENSION citus UPDATE TO '9.5-3';
+SELECT * FROM multi_extension.print_extension_changes();
```

Comment:
Honestly, I am not sure how useful these tests are since they don't print any difference even if we have an empty downgrade or a functional one. It still has some point that it shows we didn't introduce a new function etc

Reply:
I can add `select prosrc from pg_proc where proname= 'master_update_table_statistics';` to check that the source code of the function has changed accordingly.